package.json

```diff
@@ -20,7 +20,7 @@
   "dependencies": {
     "bindings": "^1.3.1",
     "nan": "^2.14.0",
-    "prebuild-install": "^5.2.2"
+    "prebuild-install": "5.2.x"
```

Comment:
Breaking `prebuild-install` commit:

https://github.com/prebuild/prebuild-install/commit/bd718e16a6ea51f6b5ea99e676a2a9e0b732d11b

Note, they dropped support for node v4 in `prebuild-install@4.0.0`.